app/javascript/articles/components/SearchSnippet.jsx

```diff
@@ -1,45 +1,25 @@
 import { h } from 'preact';
 import { articleSnippetResultPropTypes } from '../../src/components/common-prop-types';
 
-export const SearchSnippet = ({ snippetResult }) => {
-  if (snippetResult && snippetResult.body_text) {
+export const SearchSnippet = ({ highlightText }) => {
+  if (highlightText && highlightText.body_text) {
     let bodyTextSnippet = '';
 
-    if (snippetResult.body_text.matchLevel !== 'none') {
-      const firstSnippetChar = snippetResult.body_text.value[0];
+    if (highlightText.body_text[0] && highlightText.body_text[0] !== '') {
+      const firstSnippetChar = highlightText.body_text[0];
 
       let startingEllipsis = '';
       if (firstSnippetChar.toLowerCase() !== firstSnippetChar.toUpperCase()) {
         startingEllipsis = '…';
       }
-
-      bodyTextSnippet = `${startingEllipsis + snippetResult.body_text.value}…`;
-    }
-
-    let commentsBlobSnippet = '';
-
-    if (
-      snippetResult.comments_blob.matchLevel !== 'none' &&
-      bodyTextSnippet === ''
-    ) {
-      const firstSnippetChar = snippetResult.comments_blob.value[0];
-      let startingEllipsis = '';
-
-      if (firstSnippetChar.toLowerCase() !== firstSnippetChar.toUpperCase()) {
-        startingEllipsis = '…';
-      }
-
-      commentsBlobSnippet = `${startingEllipsis +
-        snippetResult.comments_blob.value}… <i>(comments)</i>`;
+      bodyTextSnippet = `${startingEllipsis +
+        highlightText.body_text.join('...')}…`;
```

Comment:
Is `join` being used now because the `body_text` begins and ends with `...`?

Reply:
body_text is now an array of two strings. Before it was only a single string bc we only took the first highlighted match and presented it. I decided to take two strings to try and give the user a little more insight into what matched for their search.